packages/rulesets/src/spectral/functions/delete-response-codes.ts

```diff
@@ -15,13 +17,38 @@ export const DeleteResponseCodes = (deleteOp: any, _opts: any, ctx: any) => {
   const path = ctx.path
   const errors = []
 
-  if (!deleteOp?.responses) {
-    return []
+  if (!deleteOp?.responses || deleteOp?.responses.length == 0) {
+    errors.push({
+      message: EmptyResponse_ERROR,
+      path: path,
+    })
+    return errors
   }
```

Comment:
I think this can be removed since I think lines 28 through 36 cover these cases
```suggestion
```

Reply:
I forgot to remove the OR condition! But I see that both conditions are covered below, now! Tx!

Reply:
removed redundant check